### DISCUSSION TITLE
Wire:ignore is not working

### DISCUSSION THREAD

**Discussion by michal-cognitive** [livewire/livewire#8294] [7. Bugs (archived)]
Every time I trigger livewire update method all <img> from layout are redownloaded:

`<ul class="grid items-start sm:grid-cols-2 lg:grid-cols-3 gap-6" wire:ignore>
        @foreach ($this->assets as $key => $media)
            <li wire:key="media-{{ $key }}" wire:ignore>
                <a href="/{{ $this->user->slug }}/{{ $media['hash'] }}" wire:ignore>
                    <img src="{{ $media['url'] }}" class="w-full h-auto" alt="Image {{ $key }}" wire:ignore>
                </a>
            </li>
        @endforeach
    </ul>`
    
    
<img width="1013" alt="image" src="https://github.com/livewire/livewire/assets/160003678/f170122d-a6a7-43db-aab4-ea825739767c">

**Comment by danie-ramdhani** [comment]
it's not because the `wire:ignore`,  it's because the url query string is changed.

livewire is fetching the new components html then morph it. so, if the new asset is change, it will download the new one (you can see the elements effect before it's morphed inside update request in components->effects->html).

![image](https://github.com/livewire/livewire/assets/58886605/55e68286-5113-4b4d-b91c-b151e476f071)


```blade
{{-- this will stay same for every single request--}}
<img src="/img/my-img.jpg?123">

 {{-- this will track the new asset every single request --}}
 <img src="/img/my-img.jpg?{{ str()->random(5) }}" />
```

**Comment by michal-cognitive** [comment]
@danie-ramdhani you are right about that but the problem is the URL of the image is always the same.
This is a signed URL with AWS, but I just compared it through 3 requests and URL is the same. URL is long but constantly the same.

<img width="574" alt="image" src="https://github.com/livewire/livewire/assets/160003678/12417fdd-01dc-4677-a2c7-ab44248e6de7">

Here is proof  that I don't have an update on the request

**Comment by jeffegiovani** [comment]
I sended a reply at [#8391 ](https://github.com/livewire/livewire/discussions/8391#discussioncomment-9909447)

Hope to help in some way
